source/features/table-input.tsx

```diff
@@ -0,0 +1,79 @@
+import './table-input.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import TableIcon from 'octicon/diff-added.svg';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '.';
+
+function generateGfmTable([w, h]: [number, number]): string {
+	return `${'|     '.repeat(w)}|\n${'|-----'.repeat(w)}|\n${`${'|     '.repeat(w)}|\n`.repeat(h)}`.slice(0, -1);
+}
+
+function addTable(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	event.preventDefault();
```

Comment:
If this is removed, the form is submitted when a grid cell is clicked. I think this is because the grid cells are `<button>`s.

But now that I think about it there's no reason for them to be buttons in the first place, other than pure semantic correctness...

Reply:
Try ;)

You're thinking of the native `type=submit` behavior, but we're using `type=button`

Reply:
> You're thinking of the native `type=submit` behavior, but we're using `type=button`

I know plain buttons aren't supposed to submit forms, but that's what happens nonetheless... 🤷

Reply:
I just tried removing that and the form is not submitted (Chrome). What browser are you in?

Reply:
Latest stable Firefox (v81) on Arch Linux.

Reply:
Tested again, it is indeed not needed. Maybe it's because I was using `form.click()` to try to close the popup.